README.md

```diff
@@ -1,2 +1,5 @@
 # The Forum Helpers
 The main site for the Scratch group "The Forum Helpers".
+
+# Want to contribute?
+First, read the [CONTRIBUTING.md](CONTRIBUTING.md) to understand this repository's guidelines.
```

Comment:
The wording here is a little funny "First read the CONTRIBUTING.md to" feels like there needs to be "file" after "CONTRIBUTING.md", or have the text to the link say something different, such as "Contributing Guidelines".

Reply:
Change it to whatever feels best to you (this is a good opportunity to try out **the dot**)

Reply:
No way! The dot works on PRs to! How is this not international news!!!!???

Reply:
I was actually kind of surprised (I tested that it did work midway through my comment, so I didn't end up being wrong)

Reply:
It seems like it only lets me review it and I don't want to break something because I haven't used the dot before, so I'm just going to change it the old fashioned way for now.

Reply:
k

Reply:
Thanks for the change!